### ISSUE TITLE
Portal v2.1.0 Upgrade Error (Duplicate Column Names) - SQLLite

### ISSUE THREAD

**Issue by bacco007** [jprochazka/adsb-receiver#214]
I've attempted to upgrade the portal using the instructions, when I open the portal i get redirected to install.php with the following error

> Error Encountered Upgrading Your ADS-B Receiver Portal
> There was an error encountered when upgrading your portal to v2.1.0.
> 
> SQLSTATE[HY000]: General error: 1 duplicate column name: aircraft

**Comment by jprochazka** [comment]
This is an issue but is not an issue.

After my absence from the project over most of the summer I merged what work I completed so far in a development branch which pertained only to the advanced portal feature so I could focus on the installation process mainly making the setup process look less archaic. Since the merge and before I could get a final 2.1.0 release launched I realized many things had changed in my absence. PiAware 3 was released which required major changes to the PiAware setup script and FlightAware changed their licensing pertaining to sharing MLAT data which required changes to the ADS-B Exchange script as well. This being said I have had to make changes over the past week or so to the master branch which was coded to be the 2.1.0 release before the 2.1.0 release was finalized.

TL;DR

This error is caused by a MySQL query trying to add a new column to a database table. If you receive this error the column already exists and the error should be safe to ignore unless after receiving the error are you able to access the portal... If this is the case let me know.

**Comment by bacco007** [comment]
I get redirected to the install page when attempting to use the portal

**Comment by jprochazka** [comment]
If you are using the Lite version:

Edit your settings.xml file and update the version.
    sudo /var/www/html/data/settings.xml
Change <version>...</version> to <version>2.1.0</version>

If you are running the advanced version:

Execute the following commands.
    MySQL -uroot -p
    USE YOUR_DATABASE_NAME;
    UPDATE adsb_settings SET Version = '2.1.0';
    exit
